compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -219,9 +220,20 @@ object Completion {
       !sym.is(allOf(Mutable, Accessor)) &&
       (
            (mode.is(Mode.Term) && sym.isTerm)
+        || (mode.is(Mode.StableTerm) && sym.isTerm && validPathSegment(sym))
```

Comment:
I don't think adding a mode is necessary, you could do `mode.is(Mode.Type) && (sym.isType || sym.isStable)` I think